### PR TITLE
455 lpr demo

### LINKED ISSUE #455: LPR Demo
Demo for license plate recognition.

### PR DESCRIPTION
**Pull request by dorgun**

### CODE REVIEW

File: samples/license_plate_recognition/README.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+```bash
+docker run --rm -it --runtime=nvidia \
+-v `pwd`/samples:/opt/savant/samples \
+-v `pwd`/data:/data:ro \
+-v `pwd`/models/license_plate_recognition:/models \
+-v `pwd`/downloads/license_plate_recognition:/downloads \
+license_plate_recognition-module \
+samples/license_plate_recognition/module_performance.yml
+```
```
Comment (abramov-oleg) [inline samples/license_plate_recognition/README.md]: Can we just reference `run_perf.sh` here, like in the other samples?

File: samples/license_plate_recognition/overlay.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        """Override draw spec for objects with label 'lpd' (license plate detection).
+        Add classifier attributes labels to the object visualisation.
+        """
+        if object_meta.label == 'lpd':
+            new_label_format = draw_spec.label.format
+
+            # one attribute per line
+            # if there's no specific attribute for the object on this frame
+            # reserve a line for it anyway
+            # so that the object's labels don't jump up and down
```
Comment (abramov-oleg) [inline samples/license_plate_recognition/overlay.py]: The comment doesn't seem to be fitting for this sample.

**Review by abramov-oleg — CHANGES_REQUESTED** [review]

File: samples/license_plate_recognition/overlay.py (hunk 2 above)
Comment (dorgun) [reply samples/license_plate_recognition/overlay.py]: fixed

File: samples/license_plate_recognition/README.md (hunk 1 above)
Comment (dorgun) [reply samples/license_plate_recognition/README.md]: replaced

**Review by abramov-oleg — APPROVED** [review]

**Pull request merged**